public/app/core/components/AccessControl/Permissions.tsx

```diff
@@ -118,7 +125,15 @@ export const Permissions = ({
   const users = useMemo(
     () =>
       sortBy(
-        items.filter((i) => i.userId),
+        items.filter((i) => i.userId && !i.isServiceAccount),
+        ['userLogin', 'isManaged']
+      ),
+    [items]
+  );
+  const serviceAccounts = useMemo(
+    () =>
+      sortBy(
+        items.filter((i) => i.isServiceAccount),
```

Comment:
```suggestion
        items.filter((i) => i.userId && i.isServiceAccount),
```

So that teams don't appear on this list.

Reply:
technically they shouldn't because isServiceAccount would be false but the double protection is nice